packages/webpack/src/ember-webpack.ts

```diff
@@ -358,9 +358,16 @@ const Webpack: PackagerConstructor<Options> = class Webpack implements Packager
       return style;
     }
 
-    const csso = await import('csso');
-    const cssContent = readFileSync(join(this.pathToVanillaApp, style), 'utf8');
-    const minifiedCss = csso.minify(cssContent).css;
+    const { transform } = await import('lightningcss');
+    const cssPath = join(this.pathToVanillaApp, style);
+    const cssBuffer = readFileSync(cssPath);
+    const { code } = transform({
+      code: cssBuffer,
+      filename: cssPath,
+      minify: true,
+      errorRecovery: true,
```

Comment:
this is inline how `csso` treats invalid syntax it just ignores it and drops the whole block from final minified output, a bit crazy I know

But given that error might be a little cryptic in embroider context (I ran into so while testing) and emberoidr/webpack is not longer suggested way it "ok" to fallback to bad behaviours